spec/lib/blog_fetcher_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe BlogFetcher do
+ describe ".feed" do
+    context "nothing cached" do
+      it "fetches requested feed" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.